libpod/runtime.go

```diff
@@ -458,7 +461,10 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 			}
 		}
 	}
-
+	runtime.config.EventsLogFilePath = filepath.Join(runtime.config.TmpDir, "events", "events.log")
```

Comment:
Why is this in NewRuntime and makeRuntime both? Drop the one in NewRuntime

Reply:
?? i dont see the one in makeRuntime. mind pointing out?

Reply:
Sorry, this should have been one line down - we're double-creating the events dir

Reply:
i have to reverse course on this.  removing this results in the test suite not having eventlog dirs being created.  there must be something about how the tmpdir is being processed.

Reply:
Wait, what?